docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -50,6 +50,15 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 == Using the authorization code flow mechanism
 
+=== Configuring the Quarkus application to redirect the user
```

Comment:
It would be useful to have this section indeed, we describe in the quickstart but sure, let's have the one in the ref doc as well. But I'd like to I propose to rework it a little bit, to avoid focusing specifically on the redirect...

How about having a section title `Enabling authorization code flow` or may be `Configuring Quarkus to support authorization code flow` ? The 2nd option is probably better as the follow up one also starts with `Configuring...`